client/src/components/Home/HomeGridEpochResults/HomeGridEpochResults.tsx

```diff
@@ -29,11 +31,17 @@ const HomeGridEpochResults: FC<HomeGridEpochResultsProps> = ({ className }) => {
   const isProjectAdminMode = useIsProjectAdminMode();
   const { data: isPatronMode } = useIsPatronMode();
 
-  const projects =
-    projectsIpfsWithRewards.map(props => ({
-      epoch,
-      ...props,
-    })) || [];
+  const projects = projectsIpfsWithRewards.reduce(
+    (acc, curr) => {
+      if (!curr.totalValueOfAllocations) {return acc;}
+      acc.unshift({
+        ...curr,
+        epoch,
+      });
+      return acc;
+    },
+    [] as (ProjectIpfsWithRewards & { epoch: number })[],
+  );
```

Comment:
I would `.filter`, `.map` and then `.reverse`, but it's just a suggestion, your solution is probably faster in O.